src/libs/share/__tests__/WebShareModal.native.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { ShareAppModal } from 'libs/share/shareApp/ShareAppModal'
+import { render } from 'tests/utils'
+
+const dismissModal = jest.fn()
+
+describe('<WebShareModal />', () => {
+  it('should not render in native', () => {
+    const renderAPI = render(<ShareAppModal visible={true} dismissModal={dismissModal} />)
+    expect(renderAPI).not.toBeInTheDocument
```

Comment:
C'est une fonction donc je pense qu'il faut l'appeler. Je pense qu'elle est plutôt utilisé avec un `getBy...` pour vérifier si l'élément est présent ou pas. Pour vérifier qu'on ne render rien tu peux faire ça :
```suggestion
    expect(renderAPI.toJSON()).toBeNull()
```